app/bootstrap.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+use Pimple\Container;
+use Symfony\Component\Console\Application;
+
+$container = new Container();
+
+$container['command.help'] = function ($c) {
+    return new \Symfony\Component\Console\Command\HelpCommand();
```

Comment:
can do. the reason for doing this is to be more in line with what you'd do with Symfony DI container where you specify the full path.